Document how to select color-mix color space (#38550)
mdn/content: files/en-us/web/css/color_value/color-mix/index.md

The **`color-mix()`** functional notation takes two {{cssxref("&lt;color&gt;")}} values and returns the result of mixing them in a given colorspace by a given amount.

> Choosing the correct color space is important for producing desired results. Given the same colors to mix, different color spaces may be more appropriate for each interpolation use case.
> 
> - In some cases, the result of physically mixing two colored lights is desired. In that case, the CIE XYZ or srgb-linear color space is appropriate, because they are linear in light intensity.
> - If colors need to be evenly spaced perceptually (such as in a gradient), the Oklab color space (and the older Lab), are designed to be perceptually uniform.
> - If avoiding graying out in color mixing is desired, i.e. maximizing chroma throughout the transition, Oklch (and the older LCH) work well for that.
> - The sRGB color space is neither linear-light nor perceptually uniform, and produces poorer results such as overly dark or greyish mixes. Only use it if you need to match the behavior of a specific device or software that uses sRGB.
> 

Reviewer: ```suggestion
Choosing the correct color space is important for producing desired results. Given the same colors to mix, different color spaces may be more appropriate depending on the interpolation use case.

- If the result of physically mixing two colored lights is desired, the CIE XYZ or srgb-linear color space is appropriate, as they are linear in light intensity.
- If colors need to be evenly spaced perceptually (such as in a gradient), the Oklab color space (and the older Lab), is appropriate as it designed to be perceptually uniform.
- If avoiding graying out in color mixing is desired, i.e. maximizing chroma throughout the transition, Oklch (and the older LCH) works well.
- Only use sRGB if you need to match the behavior of a specific device or software that uses sRGB. The sRGB color space is neither linear-light nor perceptually uniform, and produces poorer results such as overly dark or greyish mixes.

```
